src/plots/layout_attributes.js

```diff
@@ -41,6 +42,18 @@ module.exports = {
                 'by the now deprecated `titlefont` attribute.'
             ].join(' ')
         }),
+        subtitle: {
+            text: {
+                valType: 'string',
+                editType: 'layoutstyle',
+                description: 'Sets the plot\'s subtitle.'
+            },
+            font: fontAttrs({
+                editType: 'layoutstyle',
+                description: 'Sets the subtitle font.'
+            }),
```

Comment:
Do you want to add a spacing option to help increase the space between the title and subtitle?
I'm fine if you want to leave it out in case the current behavior is looking good.

Reply:
In terms of finalizing the API:
Here I start wondering of a graph that has one title and multiple subtitles e.g. to label different subplots.
If we could add a way to have multiple subtitles and position each subtitle, then it would be great. No?
If not how we are going to handle that down the road?
We could still handle that similar to what we did for legends i.e. `legend`, `legend2`, etc.
So the main question here is that should we introduce `subtitles` as an array instead of `subtitle`?

Reply:
cc: #2746.

Reply:
@archmoj We currently don't support multiple _titles_ either, right?

So I don't think it's mandatory that we support multiple subtitles right out of the gate.

In the future, I think it would be fine to add an additional property `subtitles` which accepts an array of `subtitle` items.

Reply:
Actually later we could add `title2`, `title3`, etc. similar to multiple legends & colorbars, etc. Then each `title` could have a `subtitle`.
Resolving.